gradle/plugins/src/main/kotlin/Versions.kt

```diff
@@ -0,0 +1,8 @@
+@file:Suppress("CONSTANT_UPPERCASE", "PACKAGE_NAME_MISSING")
+
+object Versions {
+    /**
+     * JDK version which is used for building and running the project.
+     */
+    const val jdk = "8"
```

Comment:
Why not `JDK`?

Reply:
a copy paste from `save-cloud`, I guess it was done to use lower cases in *.kts

Reply:
```kotlitn
kotlin {
    jvmToolchain {
        languageVersion.set(JavaLanguageVersion.of(Versions.jdk))
    }
}
```
instead of 
```kotlin
kotlin {
    jvmToolchain {
        languageVersion.set(JavaLanguageVersion.of(Versions.JDK))
    }
}
```

Reply:
leave as is to have a common approach with `save-cloud`

Reply:
ok, lgtm